TransformationSystem/Agent/TaskManagerAgentBase.py

```diff
@@ -195,6 +196,8 @@ def execute(self):
         self._addOperationForTransformations(operationsOnTransformationDict, 'submitTasks', transformations,
                                              owner=owner, ownerGroup=ownerGroup, ownerDN=ownerDN)
 
+    self.maxParametricJobs = Operations().getValue("JobScheduling/MaxParametricJobs", MAX_PARAMETRIC_JOBS)
```

Comment:
The `maxParametricJobs` should be created in the `__init__`, and maybe this line should be at the top of this method

Reply:
This module has no `__init__`...